dftimewolf/lib/collectors/osquery.py

```diff
@@ -45,11 +47,95 @@ def _ValidateOsquery(self, query: str) -> bool:
     # TODO(sydp): add more checks.
     return query.upper().startswith('SELECT ')
 
+  def _ParsePlatforms(self, platforms: str) -> List[str]:
+    """Parse and normalise the platforms value from an osquery pack.
+
+    Arguments:
+      platforms: the platforms value from an osquery pack.
+
+    Returns:
+      a list of operating system platforms.
```

Comment:
Please specify what the possible values of the list are

Reply:
Done.